include/seqan3/alignment/configuration/detail.hpp

```diff
@@ -31,6 +31,7 @@ enum struct align_config_id : uint8_t
     global,       //!< ID for the \ref seqan3::global_alignment "global alignment" option.
     local,        //!< ID for the \ref seqan3::local_alignment "local alignment" option.
     max_error,    //!< ID for the \ref seqan3::align_cfg::max_error "max_error" option.
+    method,       //!< ID for the \ref seqan3::align_cfg::method "method" option.
```

Comment:
Do you use this id somewhere?

Reply:
No, not yet, but if I understood @rrahn correctly, the idea is that it will used at later stage. Should I remove it for now?